### DISCUSSION TITLE
[QUESTION] Is -width working?

### DISCUSSION THREAD

**Discussion by KamilBaczkowski** [davatorium/rofi#1484] [Theming]
As in the title, is the `-width` option still working? I have a device with rofi 1.6.x and `-width` there works fine and also is mentioned in the manpage, but on another device with 1.7.0 it doesn't work and isn't in the manpage.

**Comment by DaveDavenport** [comment]
https://github.com/davatorium/rofi/discussions/1407

you can set width on the window widget:

```css
window {
width: 80%;
}
```
you can use percentage (%), pixels (px) or in font width/height, etc or a calculated value. (see manpage).

You can also search the [discussion](https://github.com/davatorium/rofi/discussions?discussions_q=width) forum for answer.
